stock_available_to_promise_release/tests/common.py

```diff
@@ -141,9 +142,13 @@ def _deliver(cls, picking, product_qty=None):
         if product_qty:
             lines = picking.move_ids.move_line_ids
             for product, qty in product_qty:
-                line = lines.filtered(lambda m: m.product_id == product)
-                line.qty_done = qty
+                line = lines.filtered(
+                    lambda m, product=product: m.product_id == product
+                )
```

Comment:
I'm not sure I understand this change. product is already defined, right?

Reply:
It is a fix for pre-commit: Ruff B023 warning